.github/workflows/pr_tests.yml

```diff
@@ -84,21 +84,35 @@ jobs:
       matrix:
         machines: [ 1, 2, 3, 4 ]
 
-    env:
-      BASIC_AUTH_KEY: ${{ secrets.BASIC_AUTH_KEY }}
-      BASIC_AUTH_SECRET: ${{ secrets.BASIC_AUTH_SECRET }}
-      MULESOFT_API_CIS_URL: ${{ secrets.MULESOFT_API_CIS_URL }}
-      MULESOFT_API_CIS_KEY: ${{ secrets.MULESOFT_API_CIS_KEY }}
-      MULESOFT_API_CIS_SECRET: ${{ secrets.MULESOFT_API_CIS_SECRET }}
-      MULESOFT_API_MDM_EA_URL: ${{ secrets.MULESOFT_API_MDM_EA_URL }}
-      MULESOFT_API_MDM_EA_KEY: ${{ secrets.MULESOFT_API_MDM_EA_KEY }}
-      MULESOFT_API_MDM_EA_SECRET: ${{ secrets.MULESOFT_API_MDM_EA_SECRET }}
-      GOOGLE_ANALYTICS_ID: ${{ secrets.GOOGLE_ANALYTICS_ID }}
+    # Cancel Previous runs
+    # concurrency:
+    #   group: e2e-${{ github.workflow }}-${{ github.ref }}-${{ matrix.containers }}
+    #   cancel-in-progress: true
 
     steps:
       - uses: actions/checkout@v3
       - name: Start Docker Compose
+        env:
+          API_URL: ${{ secrets.API_URL }}
+          DATABASE_URL: ${{ secrets.DATABASE_URL }}
+          MYSQL_DATABASE: ${{ secrets.MYSQL_DATABASE }}
+          MYSQL_ROOT_PASSWORD: ${{ secrets.MYSQL_ROOT_PASSWORD }}
+          SESSION_SECRET: ${{ secrets.SESSION_SECRET }}
+          BASIC_AUTH_KEY: ${{ secrets.BASIC_AUTH_KEY }}
+          BASIC_AUTH_SECRET: ${{ secrets.BASIC_AUTH_SECRET }}
+          MULESOFT_API_CIS_URL: ${{ secrets.MULESOFT_API_CIS_URL }}
+          MULESOFT_API_CIS_KEY: ${{ secrets.MULESOFT_API_CIS_KEY }}
+          MULESOFT_API_CIS_SECRET: ${{ secrets.MULESOFT_API_CIS_SECRET }}
+          MULESOFT_API_MDM_EA_URL: ${{ secrets.MULESOFT_API_MDM_EA_URL }}
+          MULESOFT_API_MDM_EA_KEY: ${{ secrets.MULESOFT_API_MDM_EA_KEY }}
+          MULESOFT_API_MDM_EA_SECRET: ${{ secrets.MULESOFT_API_MDM_EA_SECRET }}
+          GOOGLE_ANALYTICS_ID: ${{ secrets.GOOGLE_ANALYTICS_ID }}
```

Comment:
do we need this here?

Reply:
Yes these needs to be passed for E2E containers, without which containers will not spin up properly.

Reply:
hmm. this is not good. we shouldn't need the GA ID to be able to spin them up. if it doesn't exist, then the containers shouldn't crash. but maybe something to address in a different PR